server/database/queries/getDataPostQuery.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../connection');
+
+const getDataPostQuery = (searchWord) => connection.query(`SELECT * FROM PRODUCTS WHERE title LIKE'%${searchWord}%'`);
```

Comment:
!!!

Reply:
I have to take % as a string, it doesn't work when I use the => $1, [searchWord]